Patches/Core/ThingDefs_Races/Races_Mechanoid.xml

```diff
@@ -3,12 +3,12 @@
 
 	<!-- ========== Base Mechanoid ========== -->
 
-	<Operation Class="PatchOperationAdd">
+	<!--<Operation Class="PatchOperationAdd">
 		<xpath>Defs/ThingDef[@Name="BaseMechanoid"]/statBases</xpath>
 		<value>
-			<SmokeSensitivity>0</SmokeSensitivity>
```

Comment:
You've reverted a bunch of changes by not using an up-to-date version of this file